### PR TITLE
feat(back): add GetPresentationOrderResponse domain

### LINKED ISSUE #189: 登壇順序の取得

### PR DESCRIPTION
**Pull request by dino3616**
## Related Issues

- close #189 

## Reference Documents

- https://laminne.stoplight.io/docs/enshita/ac243c01e0427-

### CODE REVIEW

File: server/model/domain/get_presentation_order_response.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package domain
+
+import "time"
+
+type GetPresentationOrderResponse []struct {
+	ID       string    `json:"id"`
+	Name     string    `json:"name"`
+	Title    string    `json:"title"`
+	Duration time.Time `json:"duration"`
```
Comment (Sigumaa) [inline server/model/domain/get_presentation_order_response.go:9]:
```suggestion


type GetPresentationOrderResponse []struct {
	ID       string    `json:"id"`
	Name     string    `json:"name"`
	Title    string    `json:"title"`
	Duration string `json:"duration"`
```

**Review by Sigumaa — CHANGES_REQUESTED** [review]

File: server/model/domain/get_presentation_order_response.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package domain
+
+import "time"
+
+type GetPresentationOrderResponse []struct {
```
Comment (Sigumaa) [inline server/model/domain/get_presentation_order_response.go:5]:
```suggestion
type GetPresentationOrderResponse struct {
```

**Review by Sigumaa — CHANGES_REQUESTED** [review]

File: server/model/domain/get_presentation_order_response.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package domain
+
+import "time"
```
Comment (Sigumaa) [inline server/model/domain/get_presentation_order_response.go:3]:
```suggestion

```

File: server/model/domain/get_presentation_order_response.go (hunk 1 above)
Comment (Sigumaa) [inline server/model/domain/get_presentation_order_response.go:9]:
```suggestion
	Duration string `json:"duration"`
```

**Review by Sigumaa — CHANGES_REQUESTED** [review]

**Review by Sigumaa — DISMISSED** [review]

**Review by laminne — DISMISSED** [review]

File: server/model/domain/get_presentation_order_response.go [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package domain
+
+
+type GetPresentationOrderResponse struct {
+	ID       string    `json:"id"`
+	Name     string    `json:"name"`
+	Title    string    `json:"title"`
+	Duration string `json:"duration"`
```
[the remark this answers is not in the document]
Comment (dino3616) [reply server/model/domain/get_presentation_order_response.go:8]: 揃えたい(なんで揃ってないんだ...(困惑)

**Review by Sigumaa — APPROVED** [review]

**Review by laminne — APPROVED** [review]

**Pull request merged**